src/utils/firebase/firebase.utils.ts

```diff
@@ -0,0 +1,55 @@
+import {initializeApp} from "firebase/app"
+import {
+	createUserWithEmailAndPassword,
+	signInWithEmailAndPassword,
+	getAuth,
+	signOut,
+} from "firebase/auth"
+const firebaseConfig = {
+	apiKey: "AIzaSyBhKb0MOulldYv0eQ8Znfr5-7ng_Sh5pXQ",
```

Comment:
Do not push these important keys to remote repo ever. Create .env file and store these variables there. Import these values from there and use them.

Reply:
Done.
Created .env.local file.